src/pages/Mypage/Component/ModifyInputs/ModifyInputs.jsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const ModifyInputs = ({ profile, setMe }) => {
+  const [textLength, setTextLength] = useState(0);
+  const [changeSex, setChangeSex] = useState('');
+  const [cellphone, setCellPhone] = useState(profile.cellphone);
+  const [userName, setUserName] = useState(profile.user_name);
+  const [biography, setBiography] = useState(profile.bio);
+
+  console.log(changeSex);
+  console.log(cellphone);
+  console.log(userName);
+  console.log(biography);
+
+  const handleRadio = e => {
+    setChangeSex(e);
+  };
+
+  // const handleLength = e => {
+  //   setTextLength(e.target.value.length);
+  // };
+
+  const handleCellphone = e => {
+    setCellPhone(e.target.value);
```

Comment:
```suggestion
const {value} = e.target;
    setCellPhone(value);
```
이렇게 구조분해 할당 할 수 있습니다~

Reply:
🙌